deploy-nagios/tasks/main.yml

```diff
@@ -0,0 +1,101 @@
+---
+
+-
+  name: "Check and include productised release image versions"
+  include_vars: ../shared_files/{{ rhmap_version }}.yml
```

Comment:
This is only ran if there is an rhmap_version declared (which there wont be in digger-installer). We need it for the rhmap_ansible version